i18n/es.po

```diff
@@ -0,0 +1,12 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: i18next-conv\n"
+"POT-Creation-Date: 2019-10-10T10:41:21.864Z\n"
+"PO-Revision-Date: 2018-10-25T09:02:35.143Z\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1)\n"
+
+msgid "Click to show feedback"
```

Comment:
Is Spanish required? If so why there are so few translations in comparison with English

Reply:
Not sure if it's required, they can be easily added/revoved, but there was a fix to make in the calls to update-po task.